subworkflows/local/determine_species.nf

```diff
@@ -40,18 +41,26 @@ workflow DETERMINE_SPECIES {
 
     }else {
         log.info "Using mash screen for sample classification"
+        id_method = "Mash"
         MASH_SCREEN(contigs, params.mash.mash_sketch ? file(params.mash.mash_sketch) : error("--mash_sketch ${params.mash_sketch} is invalid"))
         results = results.mix(MASH_SCREEN.out.mash_data)
 
         parsed = PARSE_MASH(MASH_SCREEN.out.mash_data, Channel.value("top"))
         reports = reports.mix(parsed.mash_out.map{
             meta, report -> tuple(meta, params.top_hit_species, report)
         })
+
         top_hit = parsed.mash_out
         versions = versions.mix(MASH_SCREEN.out.versions)
         versions = versions.mix(parsed.versions)
     }
 
+    // Create a channle identifying pipelines output ID
```

Comment:
Change **channle** to **channel**

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/90/commits/d60495d41e4628d0233659b88f0baa0d0d50a549